web/docs/understand-vast/query-language/operators/summarize.md

```diff
@@ -20,8 +20,8 @@ suffix of a nested field instead spelling out the full name.
 - `time-resolution`: An optional duration value that specifies the rounding of
   time values in the `group-by` section. For example, `01:48` is rounded down to
   `01:00` when a 1-hour `time-resolution` is used.
-- `sum` / `min` / `max` / `any` / `all`: A list of columns to perform the
-  respective aggregation function on within the grouped buckets. Fields that
+- `sum` / `min` / `max` / `any` / `all` / `gather`: A list of columns to perform
+  the respective aggregation function on within the grouped buckets. Fields that
```

Comment:
```suggestion
  the respective aggregation function on the grouped buckets. Fields that
```

Reply:
While you're at it, I think we need to document the available aggregation functions.

Reply:
Also, this should have a real-world example for the user besides a unit test. In fact, this applies to all aggregation functions.

Reply:
From the review instructions.

> I think the docs for the summarize actions could use a lot of work, but that's for a separate PR.

Please write a separate story for this.